src/Library/demos/Box/main.blp

```diff
@@ -0,0 +1,109 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Box";
+      margin-top: 12;
+      margin-bottom: 12;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: "A widget that arranges its child widgets into a single row or column";
+      margin-bottom: 18;
+    }
+
+    Box {
+      orientation: vertical;
+
+      Box {
+        hexpand: true;
+        orientation: vertical;
+        spacing: 12;
+
+        Box {
+          halign: center;
+          spacing: 18;
+          Button button_add{
+            label: "Add Item";
+            styles [
+              "suggested-action",
+              "pill",
+            ]
+          }
+          Button button_remove{
+            label: "Remove Item";
+            styles [
+              "destructive-action",
+              "pill",
+            ]
+          }
+        }
+
+        Box {
+          halign: center;
+          spacing: 18;
+
+          CheckButton highlight {
+            label: "Highlight Box";
+          }
+
+          CheckButton toggle_orient {
+            label: "Toggle Orientation";
+          }
+
+          CheckButton homogeneous {
+            label: "Homogeneous";
+          }
+
+          CheckButton prepend {
+            label: "Prepend Items";
+          }
+        }
+
+        Label {
+          label: "Horizontal Alignment";
+        }
+
+        Scale alignment {
```

Comment:
The blue fill level here doesn't make much sense, so best just to turn it off.

```suggestion
        Scale alignment {
          has-origin: false;
```

Reply:
A `Scale` isn't really a good control for this. There are no values possible between the marks.

Same comment as above, I'd suggest using grouped toggle buttons and the exact property name / values (no upper casing)

Reply:
Also, we need 1 for halign and 1 for valign